scripts/shared/resources/nexd.yaml

```diff
@@ -0,0 +1,55 @@
+---
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: nexd
+  namespace: nexodus
+  labels:
+    name: nexd
+spec:
+  selector:
+    matchLabels:
+      name: nexd
+  template:
+    metadata:
+      labels:
+        name: nexd
+    spec:
+      restartPolicy: Always
+      nodeSelector:
+        submariner.io/gateway: \"true\"
+      hostNetwork: true
+      hostAliases:
+        # IP of the machine running the Nexodus KIND environment
+        - ip: ${HOST_IP}
+          hostnames:
+            - try.nexodus.127.0.0.1.nip.io
+            - api.try.nexodus.127.0.0.1.nip.io
+            - auth.try.nexodus.127.0.0.1.nip.io
+      tolerations:
+        - key: node-role.kubernetes.io/control-plane
+          operator: Exists
+          effect: NoSchedule
+        - key: node-role.kubernetes.io/master
+          operator: Exists
+          effect: NoSchedule
+      containers:
+        - name: nexd
+          image: quay.io/nexodus/nexd:latest
+          securityContext:
+            capabilities:
+              add: [NET_ADMIN, NET_RAW, SYS_MODULE]
+          command:
+            - nexd
+          args:
+            - --username
+            - admin
+            - --password
+            - floofykittens
+            - --insecure-skip-tls-verify
+            - router
+            - --child-prefix
```

Comment:
Now that we understood the reason for the issue, probably you can un-comment the second child_prefix below.